os_dep/linux/ioctl_cfg80211.c

```diff
@@ -3636,7 +3653,12 @@ static int rtw_cfg80211_add_monitor_if(_adapter *padapter, char *name, struct ne
 	mon_ndev->type = ARPHRD_IEEE80211_RADIOTAP;
 	strncpy(mon_ndev->name, name, IFNAMSIZ);
 	mon_ndev->name[IFNAMSIZ - 1] = 0;
-	mon_ndev->destructor = rtw_ndev_destructor;
+#if (LINUX_VERSION_CODE>=KERNEL_VERSION(4,11,9))
+		mon_ndev->needs_free_netdev = true;
```

Comment:
`false`, unless you also remove the call to `free_netdev()` from the [destructor](https://github.com/astsam/rtl8812au/blob/v5.1.5/os_dep/linux/os_intfs.c#L2996) when the kernel is 4.11.9+.